ring-core/test/ring/middleware/test/file.clj

```diff
@@ -93,3 +93,23 @@
       (is (= 200 status))
       (is (= (into #{} (keys headers)) #{"Content-Length" "Last-Modified"}))
       (is (= foo-html body)))))
+
+(deftest test-wrap-file-with-handler-first?-option
```

Comment:
Please follow the [clojure style guide](https://github.com/bbatsov/clojure-style-guide).